metrics/metrics-logger/src/main/java/org/apache/james/metrics/logger/DefaultTimeMetric.java

```diff
@@ -0,0 +1,47 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.metrics.logger;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.james.metrics.api.TimeMetric;
+
+import com.google.common.base.Stopwatch;
+
+public class DefaultTimeMetric implements TimeMetric {
+
+    private final String name;
+    private final Stopwatch stopwatch;
+
+    public DefaultTimeMetric(String name) {
+        this.name = name;
+        this.stopwatch = Stopwatch.createStarted();
+    }
+
+    @Override
+    public String name() {
+        return name;
+    }
+
+    @Override
+    public long elapseTimeInMs() {
+        return stopwatch.elapsed(TimeUnit.MILLISECONDS);
```

Comment:
Sorry but I would like to ask again. Do we need to call stop method before calculation?